pages/import_issues_page_col_set.php

```diff
@@ -125,7 +125,7 @@
 			echo sprintf( plugin_lang_get( 'column_number' ), $i + 1);
 		}
 		else {
-			echo prepare_output($t_column_title[$i]);
+			echo $t_column_title[$i];
```

Comment:
What is the reason from removing the prepare_output() which likely does the escaping?